### PR TITLE
Assignment1.2(WEB)

### PR DESCRIPTION
**Pull request by Thunderbird145**

### CODE REVIEW

**Comment by Cybotive** [comment]
I will review this

**Comment by Cybotive** [comment]
- Web project updated to output new welcome message in first pull request: ✔
- Screenshot of web browser running the Web project included in first pull request: ✔
- API project updated to output new welcome message in second pull request:  ✔
- Screenshot of web browser running the API project included in second pull request: ✔

## Note: API is first pull request for this submission and Web is second. Since there is basically no way to change this I have just put a note here instead of X's (also their commit title in the Web portion shows they may have realized this).

#10 - Link to other commit

File: SecretSanta/src/SecretSanta.Web/Startup.cs [hunk 1]
```diff
@@ -32,7 +32,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapGet("/", async context =>
                 {
-                    await context.Response.WriteAsync("Hello from Web!");
+                    await context.Response.WriteAsync("Hello from Netscape Navigator! But the browser is Google Chrome!");
```
Comment (Cybotive) [inline SecretSanta/src/SecretSanta.Web/Startup.cs:35]: Nice job!

Comment (mmwoodfo) [inline SecretSanta/src/SecretSanta.Web/Startup.cs:35]: 🥇

**Pull request closed without merging**